core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -88,7 +88,7 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     r.msg.transid.mark(
       this,
       LoggingMarkers.INVOKER_CONTAINER_START(containerState),
-      s"containerStart containerState: $containerState ($activeActivations of max $maxConcurrent) action: $actionName namespace: $namespaceName activationId: $activationId",
+      s"containerStart containerState: $containerState container: $container activations: $activeActivations of max $maxConcurrent action: $actionName namespace: $namespaceName activationId: $activationId",
```

Comment:
Not easily, because the state is not unique to the container - the state tracking is a little weird with `recreatedPrewarm` and `recreated`, which are variants of `prewarmed` and `cold`. Ideas welcome.